CHANGELOG.md

```diff
@@ -5,15 +5,30 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Legend
+# Legend
+[C] Changes is `Cere` Runtime <br/>
+[D] Changes is `Cere Dev` Runtime
 
-[C] Cere Runtime  
-[D] Cere Dev Runtime
+## [vNext + 1]
```

Comment:
Why not the `vNext`?

Reply:
Because runtime upgrades in Mainnet are blocked until we improve our backup strategies, we will work on preparing the artifacts without making the releases.